app/api/attendees.py

```diff
@@ -49,7 +49,7 @@ def before_post(self, args, kwargs, data):
             )
 
         if db.session.query(TicketHolder.id).filter_by(
-                ticket_id=int(data['ticket']), event_id=int(data['event'])
+                ticket_id=int(data['ticket']), event_id=int(data['event']), deleted_at=None
```

Comment:
Why is the need to filter the queries by `deleted_at=None` as we are already doing 
https://github.com/fossasia/flask-rest-jsonapi/commit/7b1b0865b5b2ac016333a33ab1999419a2bb13d1#diff-1c38b04e7b5726a5388ec960bb8a4bca **get_object**
https://github.com/fossasia/flask-rest-jsonapi/commit/066a050e1dcda884ecbaaa614e37fb341a1a3ef7#diff-1c38b04e7b5726a5388ec960bb8a4bca **get_collection**

Reply:
From the linked issue:
> Let's say we have an event. Now we create a tax for that event. Since tax is also an event related field,so it will also have a deleted_at column.
Now for some reason, I decide to delete the tax. Since by default we have soft deletion enabled. The tax would be soft deleted but the entry would still be in the database.
Now let's say I i.e. the event organizer wants to add tax again. So I will make another post request. Now before creating a tax object for an event, we check if a tax object already exists in the db for that event or not like this :
```
if self.session.query(Tax).filter_by(event_id=data['event']).first():
            raise ConflictException({'pointer': '/data/relationships/event'}, "Tax already exists for this event")
```
Now even though the tax entry was deleted by the user, this snippet would still raise ConflictException since that deleted entry is still present in the db.

Reply:
The library filters the `get` requests. All the changes that I have made in this PR correspond to the POST requests.